packages/db/src/config.ts

```diff
@@ -33,24 +34,39 @@ export const getSecretConnectionConfig = async (
       throw new Error(`AWS Secret ${SecretId} is missing required key '${key}'`);
     }
   });
+  const rejectUnauthorized = dbAccessMeta.rejectUnauthorized !== 'false' && dbAccessMeta.rejectUnauthorized !== false;
```

Comment:
So we don't know the type coming from the secret? 🤔 is it worth casting a string as a bool and enforcing types here rather than checking multiple options?

Reply:
We can only infer typing from what we expect the user input to be.   Users taking a json bool and turning it into a string is entirely possible.      The typing coming back from JSON.parse is always `any` also, as it's user input essentially.  

We're *functionally* typing `rejectUnauthorized` here via an explicit check. 

Doing something like 

```
Boolean(dbAccessMeta.rejectUnauthorized) 
```

won't work because 

```
> Boolean("false")
true
```

I believe there's lodash options for truthy strings, but that felt like overhead here.

Reply:
All good. Checking string and bool is fine, if a little cumbersome, but that's what we have as user input. Thanks for thinking about it.